Frontify/brand-sdk
# feat: add new color type

for supporting in guideline-blocks-settings this is needed: https://github.com/Frontify/brand-sdk/pull/744 (will also resolve failing test)

How to test:
- Use this PR web-app https://github.com/Frontify/web-app/pull/2826
- See colors in the sidebar (blocks)

Further Steps:
If this is approved merge https://github.com/Frontify/brand-sdk/pull/744 (maybe adjust it so the approved state can handle it), and sync main to dev, so ci won't fail, then merge this and release it, add new version to web-app and then its finished
